### PR TITLE
add failing code

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: molo/profiles/admin.py [hunk 1]
```diff
@@ -129,6 +129,21 @@ def get_queryset(self, request):
         return User.objects.filter(profile__site=request.site)
 
 
+class UserProfileModelAdmin(WagtailModelAdmin, ProfileUserAdmin):
+    model = UserProfile
+    menu_label = 'Permissions'
```
Comment (smarzban) [inline molo/profiles/admin.py]: @codiebeulaine please update the label here

File: molo/profiles/forms.py [hunk 2]
```diff
@@ -5,17 +5,17 @@
...
 from django.utils.translation import ugettext_lazy as _
 from django.conf import settings
 
 from wagtail.wagtailcore.models import Site
-
 from molo.profiles.models import UserProfile, UserProfilesSettings
 
 from phonenumber_field.formfields import PhoneNumberField
 
+User = get_user_model()
```
Comment (smarzban) [inline molo/profiles/forms.py:18]: @codiebeulaine why do we need to do this?

File: molo/profiles/admin.py (hunk 1 above)
Comment (codiebeulaine) [reply molo/profiles/admin.py]: @saeedmb8 what was i supposed to change it to again?

**Comment by smarzban** [comment]
👍

**Pull request merged**